docs/PAYLOAD.md

```diff
@@ -63,6 +64,59 @@ Some ways to handle this *double* event are:
 
 # Android Behaviour
 
+## Localization
+
+Plugin supported localization from resources for: title, message and summaryText.
+
+You may use simple link to locale constant.
+
+```javascript
+{
+    "registration_ids": ["my device id"],
+    "data": {
+        "title": {"locKey": "push_app_title"},
+        "message": "Simple non-localizable text for message!"
+    }
+}
+```
+
+Or use localization with formatted constants.
+
+```javascript
+{
+    "registration_ids": ["my device id"],
+    "data": {
+        "title": {"locKey": "push_app_title"},
+        "message": {"locKey": "push_message_fox", "locData": ["fox", "dog"]}
+    }
+}
+```
+
+Here is an example using node-gcm that sends the above JSON:
+
+```javascript
+var gcm = require('node-gcm');
+// Replace these with your own values.
+var apiKey = "replace with API key";
+var deviceID = "my device id";
+var service = new gcm.Sender(apiKey);
+var message = new gcm.Message();
+message.addData('title', {"locKey": "push_app_title", "locData": ["custom data"]});
+message.addData('message', 'Simple non-localizable text for message!');
+service.send(message, { registrationTokens: [ deviceID ] }, function (err, response) {
+    if(err) console.error(err);
+    else    console.log(response);
+});
+```
+
+Localization must store in strings.xml
```

Comment:
@macdonst may be like this:

You may added constants to `platforms/android/res/values/strings[-LANGUAGE].xml` manually ([Android strings](https://developer.android.com/guide/topics/resources/string-resource.html)), but if you remove or re-add platform it may drop all you changes. Or you can use hooks for add constants automatically. 

IMPORTANT: Plugin supports only string values.